src/transport/AdminPairingTable.cpp

```diff
@@ -374,6 +374,7 @@ CHIP_ERROR AdminPairingInfo::GetCredentials(OperationalCredentialSet & credentia
 
     credentials.Release();
     ReturnErrorOnFailure(credentials.Init(&certificates, 1));
+    credentialsIndex = static_cast<uint8_t>(credentials.GetCertCount() - 1U);
```

Comment:
Why is this a `uint8_t` rather than a size_t ? What if we have a device part of a large number of fabrics, and the number of credentials is > 255?

Reply:
We're using uint8_t because the return type of the GetCertCount method from OperationalCredentialSet Class is uint8_t.
I agree with your suggestion, but could we address that in a separate PR since the OperationalCredentialSet will need a complete refactor to replace all uint8_t refs to size_t?

Reply:
Also mentioned this above.  I think the idea of `OperationalCredentialSet` is really the problem.  Why do we have an object with a fixed space for `kOperationalCredentialsMax`?  What is the utility of bundling these all together like that?  An object like this can't possibly fulfill all use cases.  The only limitation for the number of supported fabrics should be memory size.

Reply:
As mentioned above, I think it should be addressed in a separate PR because the OperationalCredentialSet requires a complete refactor.